webapp/packages/plugin-sql-editor/src/SqlEditor.tsx

```diff
@@ -48,7 +48,6 @@ export const SqlEditor = observer<Props>(function SqlEditor({ state }) {
           </Pane>
         </Split>
         <SqlEditorOverlay state={state} />
-        <SqlEditorStatusBar dataSource={dataSource} />
```

Comment:
This bar can't be moved to sql editor implementation because its role is to be an abstraction for the whole sql-editor component.

For this case, it is more suitable to use `ReactCodemirrorPanel`